global_config/separation.sh

```diff
@@ -0,0 +1,7 @@
+CONFIG=separation
```

Comment:
maybe we can name the file `separation_aks` to keep inline with the other naming currently?